components/bitly/actions/create-bitlink/create-bitlink.mjs

```diff
@@ -0,0 +1,65 @@
+import { ConfigurationError } from "@pipedream/platform";
+import bitly from "../../bitly.app.mjs";
+import { formatDeepLink } from "../../common/common.utils.mjs";
+
+export default {
+  key: "bitly-create-bitlink",
+  name: "Create a Bitlink",
+  description:
+    "Converts a long url to a Bitlink and sets additional parameters.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly,
+    long_url: {
+      type: "string",
+      description: "URL to shorten",
+    },
+    title: {
+      type: "string",
+      optional: true,
+      description: "Bitlink title",
+    },
+    tags: {
+      type: "string[]",
+      optional: true,
+    },
+    domain: {
+      type: "string",
+      optional: true,
+      description: "Custom domain. e.g. bit.ly",
+    },
+    group_guid: {
+      type: "string",
+      optional: true,
+      description: "Group GUID",
+    },
+    deeplinks: {
```

Comment:
The reason I used GUID is because it is more or an acronym, and acronyms are usually camel case. A similar i have as camel case is URL. I can change that if it does not meet the standard. What do you think?